tooling/cli.js/src/helpers/rust-cli.ts

```diff
@@ -54,10 +54,11 @@ export async function runOnRustCli(
       // running local CLI since test directory exists
       const cliPath = resolve(targetPath, '../cli.rs')
       spawnSync('cargo', ['build', '--release'], cliPath)
-      const localCliPath = resolve(
-        targetPath,
-        '../cli.rs/target/release/cargo-tauri'
-      )
+      const localCliPath = process.env.CARGO_TARGET_DIR
+        ? join(process.env.CARGO_TARGET_DIR, 'release/cargo-tauri')
+        : process.env.CARGO_BUILD_TARGET_DIR
+        ? join(process.env.CARGO_BUILD_TARGET_DIR, 'release/cargo-tauri')
+        : resolve(targetPath, '../cli.rs/target/release/cargo-tauri')
```

Comment:
These are the relevant changes, other changes are: 
- from `yarn format`
- updated `Cargo.lock` and `dist` for the api example